content/recipes/pydeface_recipe.md

```diff
@@ -0,0 +1,97 @@
+## Defacing Brain MRI using PyDeface 
```

Comment:
```suggestion
---
title: PyDeface Recipe
date: 2020-12-13
tags: []
utilities: [defacing]
modalities: []
languages: [python]
software: [pydeface, FSL]

authors: [Arshitha Basavaraj]
github_handle: [Arshitha]
---

## Defacing Brain MRI using PyDeface 
```
I've added the frontmatter to the file. I've put some word, but could you go over and make sure everything is alright? You can add some more if you want.